dependencies.yaml

```diff
@@ -232,36 +231,43 @@ dependencies:
         packages:
           - wheel
           - setuptools
-  cudatoolkit:
+  cuda_version:
     specific:
       - output_types: conda
         matrices:
           - matrix:
-              cuda: "12.0"
+              cuda: "11.2"
             packages:
-              - cuda-version=12.0
-              - cuda-cudart-dev
-              - cuda-nvrtc-dev
-              - cuda-cupti-dev
+              - cuda-version=11.2
           - matrix:
-              cuda: "11.8"
+              cuda: "11.4"
             packages:
-              - cuda-version=11.8
-              - cudatoolkit
+              - cuda-version=11.4
           - matrix:
               cuda: "11.5"
             packages:
               - cuda-version=11.5
-              - cudatoolkit
           - matrix:
-              cuda: "11.4"
+              cuda: "11.8"
             packages:
-              - cuda-version=11.4
-              - cudatoolkit
+              - cuda-version=11.8
           - matrix:
-              cuda: "11.2"
+              cuda: "12.0"
+            packages:
+              - cuda-version=12.0
+  cuda:
+    specific:
+      - output_types: conda
+        matrices:
+          - matrix:
+              cuda: "12.*"
+            packages:
+              - cuda-cudart-dev
+              - cuda-nvrtc-dev
+              - cuda-cupti-dev
```

Comment:
Alphabetize.
```suggestion
              - cuda-cupti-dev
              - cuda-nvrtc-dev
```

Reply:
I'm not even sure if we need `cuda-cupti-dev` anymore. We don't enable CUPTI support in benchmarks:
- https://github.com/rapidsai/cuspatial/blob/95019825250cb3bddcffeb276f0fca39aeddaea0/conda/recipes/libcuspatial/build.sh#L5
- https://github.com/rapidsai/rapids-cmake/pull/504

Maybe we should file an issue and/or try removing it in a separate PR.